public/scripts/power-user.js

```diff
@@ -282,6 +282,7 @@ let power_user = {
     forbid_external_media: true,
     external_media_allowed_overrides: [],
     external_media_forbidden_overrides: [],
+    import_card_tags: true,
```

Comment:
No. Card tags should be an opt-in.

Reply:
But this option just makes a window pop up when importing characters. The user can choose to cancel it.

Reply:
got it, fixed